sdk/eventhub/azure-eventhub/azure/eventhub/_utils.py

```diff
@@ -232,8 +229,11 @@ def parse_sas_credential(credential):
     return (sas, expiry)
 
 
-def transform_outbound_single_message(message, message_type, to_outgoing_amqp_message):
-    # type: (Union[AmqpAnnotatedMessage, EventData], Type[EventData], Callable) -> EventData
+def transform_outbound_single_message(
+        message: Union[AmqpAnnotatedMessage, EventData],
+        message_type: Type[EventData],
+        to_outgoing_amqp_message: Callable
```

Comment:
Can Callable be any more specific?

Reply:
Yes takes in amqpannotatedmessage and returns either uamqp or pyamqp message